package.json

```diff
@@ -17,11 +17,11 @@
     "rest",
     "http"
   ],
-	"repository": {
+  "repository": {
     "type": "git",
     "url": "https://github.com/LiudasDEM/aws-lambda-middleware"
   },
-	"homepage": "https://github.com/LiudasDEM/aws-lambda-middleware#readme",
+  "homepage": "https://github.com/LiudasDEM/aws-lambda-middleware#readme",
   "author": "LiudasDEM <liudas.demikis97@gmail.com>",

```

Comment:
add yourself to contributors as per npm docs:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors